shared/AppInsightsCore/src/JavaScriptSDK/EnvUtils.ts

```diff
@@ -311,13 +311,14 @@ export function getIEVersion(userAgentStr: string = null): number {
     }
 
     var ua = (userAgentStr || "").toLowerCase();
-    if (strContains(ua, strMsie)) {
-        return parseInt(ua.split(strMsie)[1]);
-    } else if (strContains(ua, strTrident)) {
+    // Checking for Trident first since embedded IE browser control includes higher version than MSIE.
```

Comment:
This is by design as by default an embedded IE (WebBrowser) actually runs in IE7 mode, so detecting based on the Trident version would result in potentially the wrong values being used.

Returning a X-UA-Compatible met tag in your HTML (should) cause IE to update the "version" of MSIE that it's "running" as.

Reply:
We are sending a X-UA-Compatible meta tag with the following contents:
`http-equiv="X-UA-Compatible" content="IE=edge"`
  
But this is the User Agent string that we are getting back:
`Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 6.2; Win64; x64; Trident/7.0; .NET4.0C; .NET4.0E)`

It is my understanding that this should cause it to use the highest document mode supported which in this case should be IE 11 which matches up with the Trident version (7+4=11) but not the MSIE version. This is why I proposed changing it to prefer the Trident version which takes into account the X-UA-Compatible meta tag.

If the X-UA-Compatible meta tag is not used, then it would run in the default IE7 mode.